### DISCUSSION TITLE
Tables

### DISCUSSION THREAD

**Discussion by Courtny** [microsoft/atlas-design#170] [Component]
Tables are used to list all information from a data set. The base style establishes preferred padding, font-size, and font-weight treatments.

**Comment by jdanyow** [comment]
New need in Cloud Skills Challenge- table row highlighting akin to https://primer.style/css/components/box#row-themes

https://www.figma.com/file/R8JDxWktcAcOKXwPp36Z7Y/CSC-Sept?node-id=150%3A1071

**Reply by sk6786** [reply]
@Courtny

**Comment by ollips** [comment]
@Courtny  I have a question regarding the stacking on mobile design. When we've got to the point when the table's information should be stacked on mobile we relied on a specific request back then and cared about the table's main content only. And I think it's mainly because those tables didn't have headers/footers. But now I realized we have to cover these cases as well, cause we can't guarantee authors would never apply that class on tables with no header/footer. And here I need some help from the design team, cause I don't know how friendly this looks like:

<img width="910" alt="image" src="https://user-images.githubusercontent.com/57374379/151614166-1e07e096-2a78-4622-a0af-1696abd63f1d.png">

I made a mistake initially hiding header/footer rows completely but that's not right to do from the accessibility standpoint - the same info should be available all the time.

Here is a [PR](https://github.com/microsoft/atlas-design/pull/300/files) where I'm polishing table styles to match the design file provided. Please let me know what do you think or connect me with the right person to discuss it. Thanks :)
